runtimes/alibi-explain/mlserver_alibi_explain/runtime.py

```diff
@@ -87,8 +87,16 @@ async def predict(self, payload: InferenceRequest) -> InferenceResponse:
             model_name=self.name,
             model_version=self.version,
             outputs=[output_data],
+            parameters=Parameters(headers={
+                "x-seldon-alibi-type":"explanation",
+                "x-seldon-alibi-method": self.getAlibiMethod()
+            }),
         )
 
+    def getAlibiMethod(self) -> str:
+        module: str = type(self._model).__module__
+        return module.split(".")[-1]
```

Comment:
nit: if we ever change the name of the module, is that going to affect anything downstream using it? if so maybe we need to introduce a mapping?

Reply:
I'd rather not have another map of alibi module names to our names as there are so many and we want to be dynamic